torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -437,22 +434,24 @@ def test_branches(self):
 
 class TestConversion:
     def test_int_from_long(self):
+        # NB: this test assumes that the default fp type is float64
         l = [1e6, 1e12, 1e18, -1e6, -1e12, -1e18]
         li = [10**6, 10**12, 10**18, -10**6, -10**12, -10**18]
         for T in [None, np.float64, np.int64]:
             a = np.array(l, dtype=T)
             assert_equal([int(_m) for _m in a], li)
 
-        a = np.array(l[:3], dtype=np.uint64)
-        assert_equal([int(_m) for _m in a], li[:3])
```

Comment:
why are these deleted?

Reply:
It replicates the test above with `uint64`.